wgsl/index.bs

```diff
@@ -368,38 +368,45 @@ However, UTF-8 is always a valid encoding for a WGSL program.
 Note: The intent of promoting UTF-8 like this is to simplify interchange of WGSL programs
 and to encourage interoperability among tools.
 
-WGSL program text consists of a sequence of characters, grouped into contiguous non-empty sets forming:
+WGSL program text consists of a sequence of code points, grouped into contiguous non-empty sets forming:
 * [=comments=]
 * [=tokens=]
 * [=blankspace=]
 
-The program text must not include a null character.
+The program text must not include a null code point (`U+0000`).
 
-<dfn>Blankspace</dfn> is any combination of one or more of the following characters:
-* space
-* horizontal tab
-* linefeed
-* vertical tab
-* formfeed
-* carriage return
+<dfn>Blankspace</dfn> is any combination of one or more of code points from
+[=Unicode Standard Annex #31 for Unicode Version 14.0.0|Pattern_White_Space=] property.
+Following is the set of code points in [=Unicode Standard Annex #31 for Unicode Version 14.0.0|Pattern_White_Space=]:
```

Comment:
Ah, I see this is the complete list.
It took me a while, but I found the explicit list here:
http://unicode.org/Public/UNIDATA/PropList.txt  where you see Pattern_White_Space

Reply:
@dneto0 Yes, UniData contains a lot of properties, as for why I (in the OP of this PR) reference UnicodeSet is to take advantage of interactivity of it (for example, helps to quickly validate whether things collide with XID etc.)! Thanks for the merge. The OP link is (dupe from first post): https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=%5B%3APattern_White_Space%3DYes%3A%5D&g=&i=